api/proto/lightnode/lightnode.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package lightnode;
+import "common/common.proto";
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/lightnode";
+
+/////////////////////////////////////////////////////////////////////////////////////
+// Experimental: the following definitions are experimental and subject to change. //
+/////////////////////////////////////////////////////////////////////////////////////
+
+service LightNode {
+	// GetChunk retrieves a specific chunk held by the light node.
+	rpc GetChunk(GetChunkRequest) returns (GetChunkReply) {}
+
+	// StreamAvailabilityStatus streams the availability status of all chunks assigned to the light node.
+	// For use by a DA node for monitoring the availability of chunks through its constellation of agent light nodes.
+	rpc StreamAvailabilityStatus(StreamAvailabilityStatusRequest) returns (stream StreamAvailabilityStatusReply) {}
```

Comment:
Should it be `StreamAvailableChunks` or `StreamAvailableBlobs`?
The returned message is not a status (like available, not available, live, expired etc), it's just the blobs that are available at the light node.

Reply:
This is a good callout. I think there's probably still room to rename these if needed. Cody, maybe you can make a judgement call and go with it for now.

Reply:
I've changed this to `StreamChunkAvailability`. Even after this PR merges, not too late to change naming. Only becomes tricky once we have community members running light nodes of their own.